Application/EdFi.Ods.Features/OpenApiMetadata/Providers/OpenApiMetadataCacheProvider.net.cs

```diff
@@ -153,8 +153,8 @@ IEnumerable<OpenApiContent> CreateSwaggerUiSection()
                             OpenApiMetadataSections.SwaggerUi,
                             x.Key, new Lazy<string>(
                                 () =>
-                                    new SwaggerDocumentFactory(
-                                        new SwaggerDocumentContext(_resourceModelProvider.GetResourceModel())).Create(x.Value)),
+                                    new OpenApiMetadataDocumentFactory(
+                                        new OpenApiMetadataDocumentContext(_resourceModelProvider.GetResourceModel())).Create(x.Value)),
```

Comment:
lets reformat this document correctly please...

Reply:
Fixed